### PR TITLE
Us104018/alert when initial load fails

### PR DESCRIPTION
**Pull request by mtjandra**
image

### CODE REVIEW

File: components/d2l-evaluation-hub/d2l-evaluation-hub-activities-list.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ class D2LEvaluationHubActivitiesList extends mixinBehaviors([D2L.PolymerBehavior
 					</dom-repeat>
 				</d2l-tbody>
 			</d2l-table>
-			<d2l-alert id="list-alert" type="critical" hidden$="[[_health.isHealthy]]" has-close-button>
```
Comment (mtjandra) [inline components/d2l-evaluation-hub/d2l-evaluation-hub-activities-list.js]: Removed close button, does not provide much value either refresh page, or click load more again (if load more failed).

File: components/d2l-evaluation-hub/d2l-evaluation-hub-activities-list.js [hunk 2]
```diff
@@ -310,6 +313,10 @@ class D2LEvaluationHubActivitiesList extends mixinBehaviors([D2L.PolymerBehavior
 		this.set('_health', { isHealthy: false, errorMessage: 'failedToLoadMore' });
 	}
 
+	_handleInitialLoadFailure() {
+		this.set('_health', { isHealthy: false, errorMessage: 'failedToLoadData' });
```
Comment (mtjandra) [inline components/d2l-evaluation-hub/d2l-evaluation-hub-activities-list.js]: @vix633 after this PR you should be able to use `_healthy.isHealthy` related to this comment https://github.com/BrightspaceHypermediaComponents/activities/pull/76#discussion_r263444313

File: demo/d2l-evaluation-hub/d2l-evaluation-hub.html [hunk 3]
```diff
@@ -45,10 +45,20 @@
 		<div class="vertical-section-container fixedSize">
```
Comment (mtjandra) [inline demo/d2l-evaluation-hub/d2l-evaluation-hub.html]: Updated demo pages to show error on load, and give context on expectation that clicking load more second time is suppose to fail

File: components/d2l-evaluation-hub/d2l-evaluation-hub-activities-list.js [hunk 4]
```diff
@@ -264,8 +265,10 @@ class D2LEvaluationHubActivitiesList extends mixinBehaviors([D2L.PolymerBehavior
 		try {
 			var result = await this._parseActivities(entity);
 			this._data = result;
+			this._clearAlerts();
 		} catch (e) {
 			// Unable to load activities from entity.
+			this._handleInitialLoadFailure().bind(this);
```
Comment (svanherk) [inline components/d2l-evaluation-hub/d2l-evaluation-hub-activities-list.js]: I think we should call this `handleFullLoadFailure` to match `_fullListLoading`.  This won't just happen on the initial load, but also on sort and filter.

Comment (mtjandra) [reply components/d2l-evaluation-hub/d2l-evaluation-hub-activities-list.js]: Renamed

**Pull request merged**